include/Core/Engine/Engine.hpp

```diff
@@ -163,5 +163,12 @@ namespace obe::engine
         debug::Logger get_logger() const;
 
         [[nodiscard]] const vili::node& get_arguments() const;
+
+        /**
+         * \proxy{obe::engine::Engine::get_args}
+         * \rename{Args}
+         * \asproperty
+         */
+        sol::lua_value engine_get_args_proxy(const Engine* self) const;

```

Comment:
Just like here https://github.com/ObEngine/ObEngine/blob/ceeace9efa521ec6dee674548053cb0bda4b9176/include/Core/Scene/Scene.hpp#L337
Proxy functions should be defined outside of the class interface :)

Reply:
Yup, this is done through codegen using Öbidog :)
See my comment on the issue #441 it will explain all the install procedure for the codegen
If you have any issues installing Öbidog let me know !